new.go

```diff
@@ -282,13 +283,19 @@ func newBuilder(ctx context.Context, store storage.Store, options BuilderOptions
 	coptions.IDMappingOptions = newContainerIDMappingOptions(options.IDMappingOptions)
 
 	container, err := store.CreateContainer("", []string{name}, imageID, "", "", &coptions)
```

Comment:
I think the suffix here should be random as well.  Otherwise you end up with a delay on every container after the first image.

Reply:
Thanks, I completely focused on the loop below. I'll fix that.